client/trino-jdbc/src/test/java/io/trino/jdbc/JdbcDriverIT.java

```diff
@@ -33,10 +32,10 @@ public class JdbcDriverIT
             "META-INF/MANIFEST.MF",
             "META-INF/services/java.sql.Driver");
 
-    @Parameters("jdbc-jar")
     @Test
-    public void testDependenciesRelocated(String file)
+    public void testDependenciesRelocated()
```

Comment:
maybe rename this class to `TestJdbcDriverPackaging`

Reply:
This is a Maven Failsafe integration test and thus needs to end with `IT`